tests/std/tests/P0896R4_common_iterator/test.cpp

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <concepts>
+#include <iterator>
+#include <utility>

```

Comment:
Include `<type_traits>` for `is_same_v`?

Reply:
The author of P0898 really should have had `<concepts>` transitively include `<type_traits>`.